foxy/distribution.yaml

```diff
@@ -3742,6 +3742,21 @@ repositories:
       url: https://github.com/SBG-Systems/sbg_ros2.git
       version: master
     status: maintained
+  sick_safetyscanners2_interfaces:
+    doc:
+      type: git
+      url: https://github.com/SICKAG/sick_safetyscanners2_interfaces.git
```

Comment:
Consider clarifying the license identifier in the package.xml to "Apache-2.0" instead of "ALv2"